app.json

```diff
@@ -5,13 +5,25 @@
     "web": {
       "bundler": "metro"
     },
-    "name": "ke.droidcon.reactnative",
-    "slug": "ke.droidcon.reactnative",
-    "plugins": ["expo-router"],
+    "name": "DroidconKe ReactNative",
+    "slug": "droidconke-reactnative",
```

Comment:
@kharioki , the reason I had the name as `ke.droidcon.reactnative` is because these were the naming conventions given by droidconke guys.
> - We're changing the package naming so should be in this format: ke.droidcon.kotlin . the last part should be the stack eg flutter/reactnative.

Reply:
@brianwachira this is just the slug for the project name. if you check the package and bundleIdentifier I have used the `ke.droidcon.reactnative`

Reply:
oh, that's ok then